Graphs/TreeGraph/index.js

```diff
@@ -210,6 +212,20 @@ const TreeGraph = (props) => {
     const [nodeElement, setNodeElement] = useState(null);
     const [nodes, setNodes] = useState(null);
 
+    let count = 0;
+
+    const downloadTreeGraph = async () => {
+        count++;
+        await htmlToImage.toJpeg(document.getElementsByClassName('tree-graph').item(0), { quality: 0.95 })
+            .then(function (dataUrl) {
+                let link = document.createElement('a');
+                link.download = `dashboard-topology-snapshot-${count}.jpeg`;
```

Comment:
if TreeGraph was used in different places, would the download link always be dashboard-topology-snapshot(count).jpeg?  maybe the name should be a prop to this generic component